zig-aarch64.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# shellcheck disable=SC2068
+zig cc -target aarch64-linux-gnu $@
```

Comment:
```suggestion
exec zig cc -target aarch64-linux-gnu $@
```

no need to keep the shell process around in the process tree while cc runs